<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TrainGame</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>1131</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="depotPage">
     <widget class="QPushButton" name="fixButton">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>250</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:pressed {
	background-color: #222222;
	border: 1px solid #333333;
}
QPushButton:hover {
	background-color: #66db57;
}
QPushButton {
font: 75 12pt &quot;Caladea&quot;;
border: 1px solid #199909;
border-radius: 6px;
background-color: #116a06;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67, 
            stop: 0 #22c70d, stop: 1 #116a06);
}</string>
      </property>
      <property name="text">
       <string>Korjaa</string>
      </property>
     </widget>
     <widget class="QListView" name="ownedToFixListView">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>781</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="gamePage">
     <widget class="QGraphicsView" name="gameView">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>511</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="minimapView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>281</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="gasSlider">
      <property name="geometry">
       <rect>
        <x>1070</x>
        <y>300</y>
        <width>41</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider  
{  
    background-color: #ddd;   
    border-style: outset;   
    border-radius: 10px;   
}
QSlider::groove:horizontal  
{  
    height: 12px;  
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);  
    margin: 2px 0  
}  
QSlider::handle:horizontal   
{  
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 white, stop:1 green);  
    width: 16px;  
    height: 16px;  
    margin: -5px 6px -5px 6px;  
    border-radius:11px;   
    border: 3px solid #ffffff;  
}  
</string>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>99</number>
      </property>
      <property name="sliderPosition">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="directionButton">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>230</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:pressed {
	background-color: #222222;
	border: 1px solid #333333;
}
QPushButton:hover {
	background-color: #ccc5c7;
}
QPushButton {
font: 75 12pt &quot;Caladea&quot;;
border: 1px solid #72595e;
border-radius: 6px;
background-color: #605053;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67, 
            stop: 0 #bdbabb, stop: 1 #605053);
}</string>
      </property>
      <property name="text">
       <string>Suunta</string>
      </property>
     </widget>
     <widget class="QLabel" name="directionLabel">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>270</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Eteen</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="comingLabel">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>80</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tulevat</string>
      </property>
     </widget>
     <widget class="QLabel" name="passedLabel">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>80</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Menneet</string>
      </property>
     </widget>
     <widget class="QListWidget" name="nextStationsListWidget">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>110</y>
        <width>81</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="passedStationsListWidget">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>110</y>
        <width>81</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="shopPage">
     <widget class="QPushButton" name="buyButton">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>250</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:pressed {
	background-color: #222222;
	border: 1px solid #333333;
}
QPushButton:hover {
	background-color: #66db57;
}
QPushButton {
font: 75 15pt &quot;Caladea&quot;;
border: 1px solid #199909;
border-radius: 6px;
background-color: #116a06;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67, 
            stop: 0 #22c70d, stop: 1 #116a06);
}</string>
      </property>
      <property name="text">
       <string>Osta</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sellButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>250</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:pressed {
	background-color: #222222;
	border: 1px solid #333333;
}
QPushButton:hover {
	background-color: #f04d6e;
}
QPushButton {
font: 75 15pt &quot;Caladea&quot;;
border: 1px solid #b20024;
border-radius: 6px;
background-color: #ea0533;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67, 
            stop: 0 #fb2953, stop: 1 #c90029);
}</string>
      </property>
      <property name="text">
       <string>Myy</string>
      </property>
     </widget>
     <widget class="QLabel" name="playerTrainsLabel">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>0</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;Caladea&quot;;</string>
      </property>
      <property name="text">
       <string>Omat huoltojunat</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="buyableTrainsLabel">
      <property name="geometry">
       <rect>
        <x>794</x>
        <y>0</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;Caladea&quot;;</string>
      </property>
      <property name="text">
       <string>Ostettavat junat</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QListWidget" name="buyableTrainsListWidget">
      <property name="geometry">
       <rect>
        <x>835</x>
        <y>60</y>
        <width>271</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="ownedTrainsListWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>251</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="valitseJunaLabel">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>60</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Valitse tai vaihda pelattava juna</string>
      </property>
     </widget>
     <widget class="QLabel" name="costsLabelText">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>370</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Saat / Maksaa:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="confirmButton">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>120</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:pressed {
	background-color: #000080;
	border: 1px solid #333333;
}
QPushButton:hover {
	background-color: #add8e6;
}
QPushButton {
font: 75 15pt &quot;Caladea&quot;;
border: 1px solid #199909;
border-radius: 6px;
background-color: #0000ff;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67, 
            stop: 0 #0000ff, stop: 1 #0000ff);
}
</string>
      </property>
      <property name="text">
       <string>Hyväksy</string>
      </property>
     </widget>
     <widget class="QLabel" name="featuresTextOLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>440</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ominaisuudet:</string>
      </property>
     </widget>
     <widget class="QLabel" name="featuresTextBLabel">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>440</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ominaisuudet:</string>
      </property>
     </widget>
     <widget class="QLabel" name="featuresOLabel">
      <property name="geometry">
       <rect>
        <x>134</x>
        <y>440</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="featuresBLabel">
      <property name="geometry">
       <rect>
        <x>934</x>
        <y>440</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="costsLabel">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>380</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="gameButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
	background-color: #222222;
	border: 1px solid #333333;
}
QPushButton:hover {
	background-color: #66db57;
}
QPushButton {
font: 75 12pt &quot;Caladea&quot;;
border: 1px solid #199909;
border-radius: 6px;
background-color: #116a06;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67, 
            stop: 0 #22c70d, stop: 1 #116a06);
}</string>
    </property>
    <property name="text">
     <string>Pelaa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="depotButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="LinearGradientPattern">
         <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="0.670000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
          <gradientstop position="0.000000000000000">
           <color alpha="255">
            <red>34</red>
            <green>199</green>
            <blue>13</blue>
           </color>
          </gradientstop>
          <gradientstop position="1.000000000000000">
           <color alpha="255">
            <red>17</red>
            <green>106</green>
            <blue>6</blue>
           </color>
          </gradientstop>
         </gradient>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="LinearGradientPattern">
         <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="0.670000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
          <gradientstop position="0.000000000000000">
           <color alpha="255">
            <red>34</red>
            <green>199</green>
            <blue>13</blue>
           </color>
          </gradientstop>
          <gradientstop position="1.000000000000000">
           <color alpha="255">
            <red>17</red>
            <green>106</green>
            <blue>6</blue>
           </color>
          </gradientstop>
         </gradient>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="LinearGradientPattern">
         <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="0.670000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
          <gradientstop position="0.000000000000000">
           <color alpha="255">
            <red>34</red>
            <green>199</green>
            <blue>13</blue>
           </color>
          </gradientstop>
          <gradientstop position="1.000000000000000">
           <color alpha="255">
            <red>17</red>
            <green>106</green>
            <blue>6</blue>
           </color>
          </gradientstop>
         </gradient>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="LinearGradientPattern">
         <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="0.670000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
          <gradientstop position="0.000000000000000">
           <color alpha="255">
            <red>34</red>
            <green>199</green>
            <blue>13</blue>
           </color>
          </gradientstop>
          <gradientstop position="1.000000000000000">
           <color alpha="255">
            <red>17</red>
            <green>106</green>
            <blue>6</blue>
           </color>
          </gradientstop>
         </gradient>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="LinearGradientPattern">
         <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="0.670000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
          <gradientstop position="0.000000000000000">
           <color alpha="255">
            <red>34</red>
            <green>199</green>
            <blue>13</blue>
           </color>
          </gradientstop>
          <gradientstop position="1.000000000000000">
           <color alpha="255">
            <red>17</red>
            <green>106</green>
            <blue>6</blue>
           </color>
          </gradientstop>
         </gradient>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="LinearGradientPattern">
         <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="0.670000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
          <gradientstop position="0.000000000000000">
           <color alpha="255">
            <red>34</red>
            <green>199</green>
            <blue>13</blue>
           </color>
          </gradientstop>
          <gradientstop position="1.000000000000000">
           <color alpha="255">
            <red>17</red>
            <green>106</green>
            <blue>6</blue>
           </color>
          </gradientstop>
         </gradient>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="LinearGradientPattern">
         <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="0.670000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
          <gradientstop position="0.000000000000000">
           <color alpha="255">
            <red>34</red>
            <green>199</green>
            <blue>13</blue>
           </color>
          </gradientstop>
          <gradientstop position="1.000000000000000">
           <color alpha="255">
            <red>17</red>
            <green>106</green>
            <blue>6</blue>
           </color>
          </gradientstop>
         </gradient>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="LinearGradientPattern">
         <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="0.670000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
          <gradientstop position="0.000000000000000">
           <color alpha="255">
            <red>34</red>
            <green>199</green>
            <blue>13</blue>
           </color>
          </gradientstop>
          <gradientstop position="1.000000000000000">
           <color alpha="255">
            <red>17</red>
            <green>106</green>
            <blue>6</blue>
           </color>
          </gradientstop>
         </gradient>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="LinearGradientPattern">
         <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="0.670000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
          <gradientstop position="0.000000000000000">
           <color alpha="255">
            <red>34</red>
            <green>199</green>
            <blue>13</blue>
           </color>
          </gradientstop>
          <gradientstop position="1.000000000000000">
           <color alpha="255">
            <red>17</red>
            <green>106</green>
            <blue>6</blue>
           </color>
          </gradientstop>
         </gradient>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
	background-color: #222222;
	border: 1px solid #333333;
}
QPushButton:hover {
	background-color: #66db57;
}
QPushButton {
font: 75 12pt &quot;Caladea&quot;;
border: 1px solid #199909;
border-radius: 6px;
background-color: #116a06;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67, 
            stop: 0 #22c70d, stop: 1 #116a06);
}</string>
    </property>
    <property name="text">
     <string>Varikko</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shopButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed {
	background-color: #222222;
	border: 1px solid #333333;
}
QPushButton:hover {
	background-color: #66db57;
}
QPushButton {
font: 75 12pt &quot;Caladea&quot;;
border: 1px solid #199909;
border-radius: 6px;
background-color: #116a06;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67, 
            stop: 0 #22c70d, stop: 1 #116a06);
}</string>
    </property>
    <property name="text">
     <string>Kauppa</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Fame/raha:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1154</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
